DailyGoals/DailyGoals/Entities/Goal.swift

```diff
@@ -9,9 +9,35 @@ import Foundation
 
 struct Goal: Identifiable, Hashable {
 
+	enum Action: CaseIterable, CustomStringConvertible {
+		case unknown
+		case devComplete
+		case readyForQA
+		case QAcomplete
+		case done
+		case additional
+
+		var description: String {
+			switch self {
+			case .unknown:
+				return ""
+			case .devComplete:
+				return "DEV complete"
+			case .readyForQA:
+				return "Ready for QA"
+			case .QAcomplete:
+				return "QA complete"
+			case .done:
+				return "Done"
+			case .additional:
+				return "Additional"
+			}
+		}
+	}
+
 	let id = UUID()
-    var ticketNumber: String = ""
-    var description: String = ""
-    var goalAction: String?
+    var ticketNumber = ""
+    var description = ""
+	var action: Action = .unknown
```

Comment:
```suggestion
    var action: Action = .unknown
```

Reply:
Yeah we're gonna have a lot of these because Xcode on my personal laptop is set to use tabs instead of spaces for the indentation